src/lib/dnssd/minimal_mdns/responders/QueryResponder.cpp

```diff
@@ -56,6 +56,7 @@ QueryResponderSettings QueryResponderBase::AddResponder(RecordResponder * respon
     {
         return QueryResponderSettings();
     }
+    ChipLogDetail(Discovery, "Responding with %s", responder->GetQName());
```

Comment:
Does this work? I believe GetQName should return a struct like:

```
struct FullQName
{
    const QNamePart * names;
    size_t nameCount;
    // ...
}
```

Maybe skip this change from this PR just in case?

Reply:
CI says it does not work:

```
2022-06-14 23:35:24 INFO       59 |     ChipLogDetail(Discovery, "Responding with %s", responder->GetQName());
2022-06-14 23:35:24 INFO          |                              ^~~~~~~~~~~~~~~~~~~~  ~~~~~~~~~~~~~~~~~~~~~
2022-06-14 23:35:24 INFO          |                                                                       |
2022-06-14 23:35:24 INFO          |                                                                       mdns::Minimal::FullQName
2022-06-14 23:35:24 INFO    ../../src/test_driver/efr32/third_party/connectedhomeip/src/lib/support/logging/CHIPLogging.h:173:93: note: in definition of macro 'ChipLogDetail'
2022-06-14 23:35:24 INFO      173 |     chip::Logging::Log(chip::Logging::kLogModule_##MOD, chip::Logging::kLogCategory_Detail, MSG, ##__VA_ARGS__)
2022-06-14 23:35:24 INFO          |                                                                                             ^~~
2022-06-14 23:35:24 INFO    ../../src/test_driver/efr32/third_party/connectedhomeip/src/lib/dnssd/minimal_mdns/responders/QueryResponder.cpp:59:48: note: format string is defined here
2022-06-14 23:35:24 INFO       59 |     ChipLogDetail(Discovery, "Responding with %s", responder->GetQName());
```

lets revert this change.